docs/src/lib/representations.md

```diff
@@ -403,13 +403,16 @@ dim(::HPoly{Real})
 ∈(::AbstractVector{Real}, ::HPoly{Real})
 addconstraint!(::HPoly{Real}, ::LinearConstraint{Real})
 constraints_list(::HPoly{Real})
+copy(P::PT) where {N, PT<:HPoly{N}}
```

Comment:
It should be `copy(P::PT) where {PT<:HPoly{N}} where {N}`, right?

Reply:
i used the same signature as in the function, which is `copy(P::PT) where {N, PT<:HPoly{N}}`.